docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,397 @@
+            "type": "Literal",
+            "value": 2,
+            "raw": "2"
+        }
+    }
+}
+```
+
+[Static analysis](#static-analysis) tools such as ESLint typically operate by converting syntax into an AST in the ESTree format.
+
+## F
+
+### Fix
+
+An optional augmentation to a [rule](#rule) [violation](#violation) that describes how to automatically correct the violation.
+
+Fixes are generally "safe" to apply automatically: they shouldn't cause code behavior changes.
+ESLint will apply all fixes in a [report](#report) when run with the `--fix` flag.
+Fixes may also be triggered in common editor extensions.
+
+Rule violations may also include file changes that are unsafe and not automatically applied in the form of [suggestions](#suggestion).
+
+### Flat Config
+
+The current configuration file format for ESLint.
+
+The ["Legacy" config format](#legacy-config) allowed nesting configuration files in sub-directories within a project.
+"Flat" config files are named as such because all nesting must be done in one configuration file.
+
+Flat config files are named in the format `eslint.config.(c|m)?js`.
+
+You can read more about the motivations behind flat configurations in [ESLint's new config system, Part 2: Introduction to flat config](https://eslint.org/blog/2022/08/new-config-system-part-2).
+
+### Formatter (Linting)
+
+A package that presents the [report](#report) generated by ESLint.
+
+ESLint ships with several built-in reporters, including `json`, `stylish` (default), and `tap`.
+
+For more information, see [Formatters](../formatters).
+
+### Formatter (Tool)
+
+A [static analysis](#static-analysis) tool that quickly reformats code without changing its logic or names.
+
+Formatters generally only modify the "trivia" of code, such as semicolons, spacing, newlines, and whitespace in general.
+Trivia changes generally don't modify the [AST](#abstract-syntax-tree-ast) of code.
+
+Common formatters in the ecosystem include [Prettier](https://prettier.io) or [dprint](https://dprint.dev).
+
+Note that although ESLint is a [linter](#linter) rather than a formatter, ESLint still currently ships with a collection of deprecated [formatting rules](#formatting-rule).
+Formatting rules will be removed in a later major version.
+
+### Formatting (Rule)
+
+A rule that solely targets [formatting](#formatter-tool) concerns, such as semicolons and whitespace.
+These rules don't change application logic and are a subset of [Stylistic rules](#stylistic-rule).
+
+ESLint's built-in formatting rules are deprecated and will be removed in a later major version.
+ESLint recommends instead using a dedicated formatter such as [Prettier](https://prettier.io) or [dprint](https://dprint.dev).
+Alternately, the [ESLint Stylistic project](https://eslint.style) provides formatting-related lint rules.
+
+For more information, see [Deprecation of formatting rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules).
+
+## G
+
+### Global
```

Comment:
I feel like this is a bit ambiguous. Maybe we could call this "Global Variable" and explain that global variables in JavaScript are defined on the `globalThis` object by the runtime. Then maybe a separate section of "Global Declaration" where we explain how to declare globals for use in ESLint?